### ISSUE TITLE
[React Native Gallery -> Print]: Accessible name does not match with visible name for 'Print PDF' button.

### ISSUE THREAD

**Issue by YajurG** [microsoft/react-native-gallery#369]
**Pre-Requisite:**
Turn on Narrator (Ctrl + Win + Enter).

**Repro-Steps:**
Launch the React Native Gallery app.
Navigate to 'Print' button and invoke it.
Now navigate to 'Print PDF' button and listen to narrator announcement.

**Actual Result:**

Accessible name and visible name are defined incorrectly for 'Print PDF' button, when user navigated to ‘Print PDF' button, narrator is announcing as “Example, button, collapsed".

**Expected Result:**
Accessible name and visible name should be defined properly for 'Print PDF' button and when user navigates to the ‘Print PDF' button, narrator should announce as “Print PDF button".

**User Impact:**
Users with visual impairments who rely on screen reader will not be able to know the exact name of the button if accessible name and visible name are defined incorrectly, they may get confused if narrator is announcing different names.

Internal Bug Link: https://dev.azure.com/microsoft/OS/_workitems/edit/47845642
